google-cloud-spanner/src/test/java/com/google/cloud/spanner/IntegrationTestEnv.java

```diff
@@ -91,7 +91,9 @@ protected void before() throws Throwable {
     this.config.setUp();
 
     SpannerOptions options = config.spannerOptions();
-    String instanceProperty = System.getProperty(TEST_INSTANCE_PROPERTY, "");
+    // TODO: Revert back any changes to this file before merging.
```

Comment:
It seems we forgot to clean this file?

Reply:
I'll clean it before merging. Have added a TODO here.